pkg/identity/cache/allocation_test.go

```diff
@@ -599,8 +610,9 @@ func TestAllocatorReset(t *testing.T) {
 // Test that we can close and reopen the allocator successfully.
 func testAllocatorReset(t *testing.T, testConfig testConfig) {
 	labels := labels.NewLabelsFromSortedList("id=bar;user=anna")
-	owner := newDummyOwner()
-	mgr := NewCachingIdentityAllocator(owner, testConfig.allocatorConfig)
+	logger := hivetest.Logger(t)
+	owner := newDummyOwner(logger)
+	mgr := NewCachingIdentityAllocator(hivetest.Logger(t), owner, testConfig.allocatorConfig)
```

Comment:
Why do not use the logger declared above?

Reply:
My bad, we are using a new logger for the allocator I guess. :)

Reply:
ups, good catch, we should be using it.